DQMOffline/L1Trigger/python/L1TriggerDqmOfflineMC_cff.py

```diff
@@ -22,11 +22,14 @@
 
 
 # do not run the emulator client in MC
-l1TriggerDqmOfflineClient.remove(l1EmulatorMonitorClient) 
+l1TriggerDqmOfflineClient.remove(l1EmulatorMonitorClient)
 
+import DQMOffline.L1Trigger.L1TriggerDqmOffline_cff as stage2_cffs_
+stage2_cffs_.Stage2l1TriggerDqmOffline.remove(stage2_cffs_.l1tStage2EmulatorMonitorClient)
```

Comment:
Hi,@davidlange6. I don't think so. We put it in L1TriggerDqmOfflineMC_cff because we want to keep 'l1tStage2EmulatorMonitorClient' in data running but not in MC.